Instruments/config/2016/Skimmer.cfg

```diff
@@ -96,20 +86,19 @@ file: TTTo2L2Nu_TuneCUETP8M2.root
 [TTToSemiLeptonic]
 weights: TopPt
 merged_output: TTToSemiLeptonic.root
-file: TTToSemilepton_TuneCUETP8M2.root TTToSemiLeptonic_TuneCP5.root
+file: TTToSemilepton_TuneCUETP8M2.root
 
 [TTToHadronic]
 weights: TopPt
 merged_output: TTToHadronic.root
-file: TTToHadronic.root
-file: TTToHadronic_PSweights.root
+file: TTToHadronic_TuneCP5.root
```

Comment:
why for  TTToHadronic you are using TuneCP5 while for the others you are using TuneCUETP8M2?